pkg/handlers/dashboard.go

```diff
@@ -35,28 +35,30 @@ func (h *Handler) GetAppDashboard(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	a, err := store.GetStore().GetAppFromSlug(appSlug)
+	store := store.GetStore()
```

Comment:
i'd suggest avoiding variable shadowing. if you really wanna have this in a separate step, maybe change the variable name to something else

Reply:
reverted this change and will just pass `store.GetStore()` to the `GetGraphs(...)` function